src/v2/Apps/Order/Utils/orderUtils.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  CommercePaymentMethodEnum,
+  Payment_order,
+} from "v2/__generated__/Payment_order.graphql"
+
+export const isPaymentValid = (
```

Comment:
#non-blocking. `isPaymentSet` reads better to me because we're not really validating payment.